app/database/migrations/2015_11_05_183749_image.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class Image extends Migration {
+
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table('assets', function(Blueprint $table)
+		{
+			$table->text('image')->after('notes')->nullable();
+			//
+		});
+	}
+
+	/**
+	 * Reverse the migrations.
+	 *
+	 * @return void
+	 */
+	public function down()
+	{
+		Schema::table('assets', function(Blueprint $table)
+		{
+			//
```

Comment:
Need to remove the field in the `down` section. Otherwise if someone rolls back a migration, it will break when they try to re-run, since the field will already exist.